app/webpacker/components/RegistrationsV2/Register/index.jsx

```diff
@@ -122,7 +107,6 @@ function Register({
             preferredEvents={preferredEvents}
             competitionInfo={competitionInfo}
             registration={registration}
```

Comment:
StepPanel could access registration itself via the hook.

Reply:
it actually does, just didn't remove that